<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Social Navigation Dataset Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>800</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>800</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label5">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>perfect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label4">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>very good</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label3">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>good</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label2">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>acceptable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label1">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>undesirable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label0">
              <property name="styleSheet">
               <string notr="true">color: rgba(0, 0, 0, 0.);</string>
              </property>
              <property name="text">
               <string>unacceptable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="slider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>send
context
assessment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getButton">
          <property name="text">
           <string>get new
sample</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="estimateBox">
          <property name="text">
           <string>automatically
estimate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="estimateButton">
          <property name="text">
           <string>estimate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
